client/core/core.go

```diff
@@ -1085,6 +1088,24 @@ func (c *Core) Trade(pw string, form *TradeForm) (*Order, error) {
 		return nil, fmt.Errorf("unknown DEX %s", form.DEX)
 	}
 
+	dc.suspensionMtx.RLock()
+	suspensionEpoch := dc.suspensionEpoch
+	dc.suspensionMtx.RUnlock()
+
+	// TODO: (Refinement) instead of outrightly avoiding placing orders when
+	// a trade suspension is received, we need to decide whether a trade
+	// can be taken based on the block times of the blockchains involved
+	// in the trade.
+	//
+	// timeRemaining := time.Duration(int64(suspensionEpoch) - time.Now().Unix())
```

Comment:
Even though this is commented, this formula is incorrect. Epoch must be multiplied by epoch duration to get a time.